dev_guide/routes.adoc

```diff
@@ -164,3 +164,47 @@ console or using `oc get routes` displays the host and certificates specified by
 the user. However, custom hosts are not respected, and routes with custom
 certificates do not work.
 endif::[]
+
+[[routes-load-balancing-for-AB-testing]]
+== Load Balancing for A/B Testing
+You can run two versions of an application and control,
+entirely within {product-title}, the percentage of traffic to and from each
+application for A/B testing.
```

Comment:
I think the above gets a little confusing with the "application and control" bit. It's like it's the two versions you can run, but gets confusing until you realise the aside within the commas is an aside, rather than the secnod part of the sentence. Maybe:

"You can run two versions of an application, and, entirely within {product-title}, control the percentage of traffic to and from each application for A/B testing."

It's not pretty, but it makes the most sense :|